### PR TITLE
Create ByteSizeOfString.md

### PR DESCRIPTION
**Pull request by shafa-shabudeen**
Created a program that demonstrates how to calculate the size of a string in bytes. It takes a user-defined string, converts it to bytes using the default character encoding (usually UTF-8), and then returns the size of the string in bytes

### CODE REVIEW

File: src/ByteSizeOfString.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+## Snip Langauge
+* [Java](#java)
+* [Python](#python)
+* [JavaScript](#javascript)
+
+### Java
+```java
+  public static void lengthOfbyteString(String str){
+        int sizeInBytes = str.getBytes().length;
+        System.out.println("Size of the string in bytes: " + sizeInBytes);
```
Comment (rahulrk-dev) [inline src/ByteSizeOfString.md:18]: Do not just print the result, return them for all 3 languages

**Review by rahulrk-dev — CHANGES_REQUESTED** [review]
Use camel casing for file name

**Comment by shafa-shabudeen** [comment]
I made the changes and rename the file also.

**Review by rahulrk-dev — APPROVED** [review]
Thanks @shafa-shabudeen, merging

**Comment by shafa-shabudeen** [comment]
Thank you

**Pull request merged**